activation/activation_test.go

```diff
@@ -101,7 +99,7 @@ func (*postProverClientMock) SetLogger(shared.Logger) {}
 func (*postProverClientMock) SetParams(datadir string, space uint64) error { return nil }
 
 func (*postProverClientMock) Cfg() *config.Config { return nil }
-
+*/
```

Comment:
it's test code, but still, the comments and leftovers might be confusing since you have this code in a different file.